### ISSUE TITLE
[Accessibility] Add page up/down support for accessible buffer

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#173452]
CC @meganrogge and @Tyriar 

I have built and tested the latest main branch and discovered the following issue.

When there is much more information displayed in a11y terminal buffer than its default aria document role buffer size, pressing Ctrl+Home key in the terminal buffer does not necessarily jump to the very first line.

I pressed `Shift+PageUp` in terminal input to scroll up the buffer content, but it did not work.

I don't know if this is possible, but it would be even nicer if arrow keys in terminal buffer can be synced with scroll keys for screen reader users.

FYI, I have tested the latest build with NVDA on Windows 11.

**Comment by meganrogge** [comment]
When I use arrow keys, it does scroll for me, but I don't see voiceover reading anything.

**Comment by meganrogge** [comment]
On windows with NVDA, using `NVDA key + up and down arrow` works for me when I've focused the accessible buffer. 

Also, pressing `Ctrl+Home` and `Ctrl+End` do jump to the top and bottom line for me.

**Comment by meganrogge** [comment]
> I pressed Shift+PageUp in terminal input to scroll up the buffer content, but it did not work.

Can confirm this does not work, so we can consider supporting that

**Comment by Tyriar** [comment]
@jooyoungseo we just changed it to be role=document and no longer content editable so it should act more like a webpage. It appeared to be accessible from out testing last week, FYI we cannot use the textarea approach that was used previously as we plan on adding semantics so you can navigate between commands via headers https://github.com/microsoft/vscode/issues/173067

**Comment by meganrogge** [comment]
we're adding a setting for content editable in case users prefer that experience https://github.com/microsoft/vscode/pull/173322

**Comment by Tyriar** [comment]
@meganrogge I think this works now since it's using monaco?

**Comment by meganrogge** [comment]
Nice, you're correct - when I had last tried, this didn't work likely because of some bug that has since been fixed
